### ISSUE TITLE
Main Engine Crash

### ISSUE THREAD

**Issue by klrkarate** [bluemagic123/tModLoader#3459]
### Version

Latest 1.4-stable

### OS

Windows

### Platform

Steam

### This bug affects

Gameplay as a Player

### Description

Upon startup of the game I get this crash report. Uninstalling terraria or TModLoader and verifying their game files have not fixed this issue and all of my drivers are up to date. Base game terraria runs fine.

### Log File

![tmodloader crash report](https://user-images.githubusercontent.com/132787562/236647481-376db359-458e-4fe0-97f3-73b9e15f1350.JPG)

[tModLoader-Logs.zip](https://github.com/tModLoader/tModLoader/files/11413434/tModLoader-Logs.zip)

### Steps to reproduce

Start TModLoader
It crashes after a few seconds trying to load.

### Expected Behavior

The game should be launching

### Actual Behavior

The game is crashing

### Reproduction frequency

100% of the time

**Comment by JavidPack** [comment]
Duplicate of https://github.com/tModLoader/tModLoader/issues/3456, please post there.

**Comment by Exhaustiver** [comment]
NOT A FIX

I also have this bug, i've spent around 2 hours playing around with the files and i found out that my problem wasnt in the mod files but was in the player files, i went to Documents/My Games/Terraria/TModLoader/Players and i moved all the player files to a seperate backup folder. I launched the game and my characters wasnt there (as expected) but when i tried to make a new one it crashed again. After a while I relaunched it and i tried reloading the mods in first by going to workshop and disabling and enabling all the mods again, and i was able to make a new character. But after making the character and quitting and relaunching the game, it crashed again

i hope this comment could somehow make someone find out why this is a bug and how to fix it because after that point i just gave up.
